pyproject.toml

```diff
@@ -427,3 +400,14 @@ force-single-line = true
 [tool.ruff.lint.pyupgrade]
 # Preserve types, even if a file imports `from __future__ import annotations`.
 keep-runtime-typing = true
+
+[tool.tomlsort]
+in_place = true
+# Sorting options:
+no_sort_tables = true
+sort_inline_arrays = true
```

Comment:
If it's desirable to have separate PRs, one for formatting and another for sorting, we can remove `sort_inline_arrays = true` and add this line in a separate PR